### ISSUE TITLE
About mpirun in running mmseqs search 

### ISSUE THREAD

**Issue by DaMaoShan** [soedinglab/MMseqs2#213]
mpirun is a kind of linux command.
But it has two kinds of very different version:  OpenMPI ,    MPICH

A bug when I run a shell scripts including "mmseqs search" in linux system which has installed mpirun of version:  (Open MPI) 1.10.2
No bug when I run a shell scripts including "mmseqs search" in linux system which has installed mpirun of version:  3.3a2  (After googled this number, I found this is from MPICH)

The bug reports:
Corrupt memory, too many entries!

Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.

mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

Process name: [[38952,1],10]
Exit code:    1

Error: Prefilter died

So could you tell me which mpirun is the base of mmseqs ?
Or How to do with this bug?
Thank you very much in advance!

**Comment by milot-mirdita** [comment]
Could you please post the commands you executed and the full output?

**Comment by DaMaoShan** [comment]
> Could you please post the commands you executed and the full output?

It is too long and too disordered to be posted.
Let me try again.
Thanks

**Comment by milot-mirdita** [comment]
You can use https://gist.github.com to post the log. 

Alternatively, the call to MMseqs2 alone would already help with diagnosing the issue.

**Comment by DaMaoShan** [comment]
> You can use https://gist.github.com to post the log.
> 
> Alternatively, the call to MMseqs2 alone would already help with diagnosing the issue.
Thank you very much for your reply.
I just solved the bug very fortunatelly
The method is to install mpich 
Download hydra-3.3.1(https://www.mpich.org/downloads/)
./configure --prefix=$DIR/mpich
make
make install
add the mpirun in the end of your .bashrc

Best wishes
